src/js/controller/controller.js

```diff
@@ -606,14 +604,17 @@ Object.assign(Controller.prototype, {
         }
 
         function _seek(pos, meta) {
-            if (_model.get('state') === STATE_ERROR) {
+            const state = _model.get('state');
+            if (state === STATE_ERROR) {
                 return;
             }
             _programController.position = pos;
-            if (!_model.get('scrubbing') && _model.get('state') !== STATE_PLAYING) {
+            if (!_model.get('scrubbing') && state !== STATE_PLAYING) {
                 meta = meta || {};
                 meta.startTime = pos;
-                this.play(meta);
+                if (state === STATE_COMPLETE || _model.get('mediaModel').get('duration') - pos < 1) {
```

Comment:
is this the desired behavior for complete ? Why would it be different than idle ?

Reply:
You can definitely do `_model.mediaModel`.